dotCMS/src/main/java/com/dotmarketing/fixtask/tasks/FixTask00085FixEmptyParentPathOnIdentifier.java

```diff
@@ -0,0 +1,100 @@
+package com.dotmarketing.fixtask.tasks;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.dotmarketing.beans.FixAudit;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.fixtask.FixTask;
+import com.dotmarketing.portlets.cmsmaintenance.ajax.FixAssetsProcessStatus;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.MaintenanceUtil;
+
+public class FixTask00085FixEmptyParentPathOnIdentifier implements FixTask {
+
+    private int total = 0;
+
+    @Override
+    public List<Map<String, Object>> executeFix() throws DotDataException, DotRuntimeException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.exception.DotRuntimeException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)